### PR TITLE
test: fix test_ioctx_context_manager raise ObjectNotFound

### PR DESCRIPTION
**Pull request by zhengchengyao**
Signed-off-by: Zheng Yin <zhengyin@cmss.chinamobile.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by zy751713126** [comment]
steps:
sh /data/ceph/qa/workunits/rados/test_python.sh

test_rados.test_rados_init_error ... ok
test_rados.test_rados_init ... ok
test_rados.test_ioctx_context_manager ... ERROR
test_rados.test_parse_argv ... ok
test_rados.test_parse_argv_empty_str ... ok

======================================================================
ERROR: test_rados.test_ioctx_context_manager
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/data/ceph/src/test/pybind/test_rados.py", line 37, in test_ioctx_context_manager
    with conn.open_ioctx('rbd') as ioctx:
  File "rados.pyx", line 579, in rados.requires.wrapper.validate_func (/data/ceph/build/src/pybind/rados2/pyrex/rados.c:7                                                                                532)
  File "rados.pyx", line 1300, in rados.Rados.open_ioctx (/data/ceph/build/src/pybind/rados2/pyrex/rados.c:16115)
ObjectNotFound: [errno 2] RADOS object not found (error opening pool 'rbd')

----------------------------------------------------------------------
Ran 72 tests in 362.181s

FAILED (errors=1)


Cluster doesn't create "rbd" pool default,  So, we have to create "rbd" pool firstly.

**Comment by zy751713126** [comment]
jenkins test make check

**Comment by zy751713126** [comment]
jenkins test make check

**Comment by zhengchengyao** [comment]
@dillaman please take a review, thanks

**Comment by zhengchengyao** [comment]
jenkins test make check

File: src/test/pybind/test_rados.py [hunk 1]
```diff
@@ -34,8 +34,11 @@ def test_rados_init():
 
 def test_ioctx_context_manager():
     with Rados(conffile='', rados_id='admin') as conn:
+        if not conn.pool_exists('rbd'):
```
Comment (tchaikov) [inline src/test/pybind/test_rados.py]: could you help me understand why we could run into this issue given that "rbd" pool is created by https://github.com/ceph/ceph/blob/96b3a8a5ee359a235188866cf8fe11af33253ff7/qa/workunits/rados/test_python.sh#L3 ?

Comment (zhengchengyao) [reply src/test/pybind/test_rados.py]: Hi @tchaikov , I find that "ceph osd pool create rbd"  was added in Nov 13  2019. I run into this issue in Nov 7 2019. 
The issue was resolved, I will close this PR.  thanks.

Comment (tchaikov) [reply src/test/pybind/test_rados.py]: @zhengchengyao thank you!

**Pull request closed without merging**